### ISSUE TITLE
Webpack 2 breaks ripcord

### ISSUE THREAD

**Issue by ggascoigne** [cdaringe/ripcord#150]
Recent create-react-app projects use webpack 2, this has a radically different webpack config format.

Both `ripcord report` and `ripcord licenses` blow up in that environment:

```
cross-env NODE_ENV=production ripcord report --webpack-config node_modules/react-scripts/config/webpack.config.prod.js -o build/reports/project/npm-dependencies.json
Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration has an unknown property 'json'. These properties are valid:
   object { amd?, bail?, cache?, context?, dependencies?, devServer?, devtool?, entry, externals?, loader?, module?, name?, node?, output?, performance?, plugins?, profile?, recordsInputPath?, recordsOutputPath?, recordsPath?, resolve?, resolveLoader?, stats?, target?, watch?, watchOptions? }
   For typos: please correct them.
   For loader options: webpack 2 no longer allows custom properties in configuration.
     Loaders should be updated to allow passing options via loader options in module.rules.
     Until loaders are updated one can use the LoaderOptionsPlugin to pass these options to the loader:
     plugins: [
       new webpack.LoaderOptionsPlugin({
         // test: /\.xxx$/, // may apply this only for some modules
         options: {
           json: ...
         }
       })
     ]
error Command failed with exit code 1.
```

**Comment by cdaringe** [comment]
i need to run to the airport and will be in late tomorrow, but, i took a first stab. a new build should kick out soon.  i'll definitely test tomorrow, so dont feel obligated to be the guinea pig.  just FYI!

**Comment by cdaringe** [comment]
Fixed in 3.0.1
